PocketKit/Sources/PocketKit/Main/MainView.swift

```diff
@@ -92,29 +85,16 @@ public struct MainView: View {
         .sheet(isPresented: $model.isPresentingHooray) {
             PremiumUpgradeSuccessView()
         }
-    }
-
-    func makeUIKitHome() -> some View {
-        HomeViewControllerSwiftUI(model: model.home)
-            .edgesIgnoringSafeArea(.all) // Allow Home to use the entire screen, including under the status bar
-            .tabBarHeightOffset { offset in tabBarHeightOffset = offset }
-            .tabItem {
-                if model.selectedSection == .home {
-                    Image(asset: .tabHomeSelected)
-                } else {
-                    Image(asset: .tabHomeDeselected)
-                }
-                Text(Localization.home)
-            }
-            .accessibilityIdentifier("home-tab-bar-button")
-            .tag(MainViewModel.AppSection.home)
-            .task {
-                Log.breadcrumb(category: "SWiftUIHome", level: .debug, message: "UIKit Home initialized.")
+        .fullScreenCover(isPresented: $model.isPresentingWebView) {
```

Comment:
by removing UIKit home, some things needed to he handled differently, and in particular the presentation of an item from a deep link that needs a web view goes here.